src/exampleTypes/ResultTable/Header.js

```diff
@@ -27,6 +27,14 @@ const moveColumn = (
   })
 }
 
+const addColumn = (mutate, computeSideIndex, includes, triggerField, field) => {
+  let index = includes.indexOf(triggerField)
+  if (index >= 0) {
+    includes.splice(computeSideIndex(index), 0, field)
```

Comment:
`F.insertAtIndex` will clean this up

Reply:
That breaks `mobx` and it does not work.